wordpress-mobile/WordPressKit-iOS
# Issue/4 removing af networking part 3

This PR update WordPressComAuthClient to use AlamoFire instead of AFNetworking.

To make sure the behaviour is consistent unit tests where added for the various OAuth scenarios to make sure that before/after behaviour is the same.

How to test:
 - Make sure unit tests are passing
 - Check on the main app, the login flows, regular user and signup using Google SignIn. Use the branch of this [PR on the main app](https://github.com/wordpress-mobile/WordPress-iOS/pull/9523) to test
